contents/customers/ycombinator.md

```diff
@@ -0,0 +1,48 @@
+
+“We use PostHog for [Startup School](https://www.startupschool.org/), our public [YC Startup Library](https://www.ycombinator.com/library), and for [Co-Founder Matching](https://www.ycombinator.com/cofounder-matching),” said Cat Li, Product and Engineering Lead for Y Combinator Startup School. “PostHog is especially useful for Co-Founder Matching, which is essentially a dating app for co-founders.”
+
+### How Y Combinator gathers 30% more data with PostHog than Google Analytics
+Y Combinator empowers each team to choose its own processes, tools and OKRs. Some teams use tools like Segment or Amplitude — but for the flagship Startup School project, Cat's team decided PostHog was the best tool for the job. 
+
+“Startup School is hugely popular, so we thought a lot about what tools to use,” said Cat. “Many platforms we looked at, including Google Analytics, dropped 30% of user data due to adblockers, or third-party cookies. PostHog became the obvious choice because it didn't have those issues.”
+
+PostHog also offered the benefit of autocapture, which enabled the team to get started quickly without wasting weeks defining tracking plans. 
+
+“Tools we looked at either weren't robust enough, or required a huge amount of work to instrument,” said Cat. “But with PostHog, we could autocapture most events using [the JS snippet](/docs/integrate/client/snippet-installation) and still [configure custom events](tutorials/event-tracking-guide) when we needed to. It was the perfect balance.”
+
+### How Y Combinator boosted Co-founder Matching engagement by 40% 
+Since getting started with PostHog, Y Combinator has gone beyond analytics and begin using other PostHog features, such as [experiments](/product/experimentation) and [session recordings](/product/session-recording), to explore data from multiple angles at once.
+
+For Startup School, for example, the team uses [trends insights](/manual/trends) to monitor weekly users and share results in [dashboards](/manual/dashboards) — while [experiments](/manual/experimentation) have led to several significant improvements for Co-founder Matching. 
+
+“PostHog's experimentation suite is really great,'” said Cat. “We recently used it to improve our matching algorithm by running an experiment which hides profiles that have been stale for 3, 6, 9 or 12 weeks. We found that users in the 6-week group sent 40% more messages than the control group - a huge improvement for us!”
+
+“And, perhaps more importantly, this experiment group had 35% more of their requests accepted - which translates to having 35% more matches than the control group.”
+
+Experiments and session recordings have also helped Y Combinator avoid pitfalls borne from misleading feedback.
```

Comment:
Maybe a second sentence here to fill out this point a little more. Something about preventing wasted resources?

Reply:
```suggestion
Experiments and session recordings have also helped Y Combinator avoid pitfalls borne from misleading feedback. As a result, PostHog ensures the team continues to add real value for users.
```